test/iron-menubar-behavior.html

```diff
@@ -96,6 +109,43 @@
           }, 200);
         });
 
+        suite('left / right keys are reversed when the menubar has RTL directionality', function() {
+          var LEFT = 37;
+          var RIGHT = 39;
+
+          test('left key moves to the next item', function(done) {
+            var rtlContainer = fixture('rtl');
+            var menubar = rtlContainer.querySelector('test-menubar');
+            menubar.selected = 0;
+            menubar.items[1].click();
+            setTimeout(function() {
+              MockInteractions.pressAndReleaseKeyOn(menubar, LEFT);
+              setTimeout(function() {
+                assert.equal(document.activeElement, menubar.items[2], 'document.activeElement is next item');
+                done();
+              // wait for async in _onFocus
+              }, 200);
```

Comment:
I think I copied this from the other examples with the assumption that both actions would need to wait for the same thing because they both affect focus, so I wouldn't be surprised if the timing is too loose. I'll look more into this.